### DISCUSSION TITLE
Come up with a name for our application!

### DISCUSSION THREAD

**Discussion by surajkumar** [javadiscord/java-discord-api#54] [Ideas]
We need a project name for our cool library!

Drop suggestions and we can do a poll.

**Comment by thegatesdev** [comment]
Accord, as in the opposite of discord.
Or some variation like AccordAPI.

**Comment by surajkumar** [comment]
disbot

**Comment by surajkumar** [comment]
an acronym using the first letter of each of the v1 contributors

**Comment by surajkumar** [comment]
java discord

**Comment by surajkumar** [comment]
better-jda
better-than-jda

**Comment by tferdous17** [comment]
Discava

"Discord" merged with "Java"
